<launch>
	
	<!-- WIIMOTE NODE -->
	<node respawn="true" pkg="wiimote"  type="wiimote_node.py" name="Wiimote_start" />	
	
	<!-- PS3 JOY NODE -->
	<node respawn="true" pkg="joy" type="joy_node" name="ps3_joy">
		<param name="dev" type="string" value="/dev/input/js1" />
		<param name="deadzone" value="0.12" />
	</node>

 	<!-- Axes -->
  		<param name="axis_linear" value="1" type="int"/>
  		<param name="scale_linear" value="2" type="double"/>

	<!-- Starting the test node -->
	<node pkg="ross_gcer_study" type="try_test_pswii" name="pioneer" />

	<!-- RosAria NODE -->
	<!-- node pkg="rosaria" type="RosAria" name="RosAria" -->
		<!-- param name="port" value="/dev/ttyUSB0" / -->
	<!-- /node -->

</launch>
